src/world.rs

```diff
@@ -407,11 +407,12 @@ impl<L> LazyInsertInternal for L
 /// component storages have to be borrowed mutably.
 ///
 /// This resource is added to the world by default.
-pub struct LazyInsertions {
-    stack: TreiberStack<Box<LazyInsertInternal>>
+pub struct LazyUpdate {
+    insert_stack: TreiberStack<Box<LazyInsertInternal>>,
```

Comment:
Let's also rename `LazyInsertInternal` to `LazyUpdateInternal` so we do not need two stacks.